README.md

```diff
@@ -38,70 +50,89 @@ Take a look at this code (adapted from our first [tutorial](https://github.com/r
 
+def sd_generate(prompt):
+    model = StableDiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-2-base").to("cuda")
+    return model(prompt).images[0]
 
 if __name__ == "__main__":
-    gpu = rh.cluster(name='rh-v100', instance_type='V100:1', provider='gcp')
-    generate_gpu = rh.function(fn=sd_generate).to(gpu, reqs=['./', 'torch==1.12.0', 'diffusers'])
-
-    images = generate_gpu('A digital illustration of a woman running on the roof of a house.', num_images=2, steps=50)
-    [image.show() for image in images]
+    gpu = rh.cluster(name='my-a100', instance_type='A100:1', provider='cheapest')
+    sd_generate = rh.function(sd_generate).to("my-a100", reqs=['./', 'torch', 'diffusers'])
+    sd_generate("An oil painting of Keanu Reeves eating a sandwich.").show()
 
-    generate_gpu.save(name='sd_generate')
+    sd_generate.save(name='sd_generate')
 ```
 By saving, I or anyone I share with can load and call into this service with a single line of code, from anywhere
 with a Python interpreter and internet connection (notebook, IDE, CI/CD, orchestrator node, etc.):
 ```python
-generate_gpu = rh.function(name='sd_generate')
-images = generate_gpu("A hot dog made of matcha.")
+import runhouse as rh
+
+sd_generate = rh.Function.from_name('sd_generate')
+image = sd_generate("A hot dog made of matcha.")
 ```
 There's no magic yaml, DSL, code serialization, or "submitting for execution." We're
 just spinning up the cluster for you (or using an existing cluster), syncing over your code,
-starting a gRPC connection, and running your code on the cluster.
-
+starting a gRPC connection, and running your code on the cluster. 
 **_Runhouse does things for you that you'd spend time doing yourself, in as obvious a way as possible._**
 
-And because it's not stateless, we can pin the model to GPU memory, and get ~1.5s/image
-inference before any compilation.
-
 On the data side, we can do things like:
 
 ```python
 # Send a folder up to a cluster (rsync)
-rh.folder(path=input_images_dir).to(system=gpu, path="dreambooth/instance_images")
+rh.folder(path="./instance_images").to(system=gpu, path="dreambooth/instance_images")
+gpu.run([f"accelerate launch diffusers/examples/dreambooth/train_dreambooth.py"
+         f"--instance_data_dir=dreambooth/instance_images "
+         f"--output_dir=dreambooth/output "])
 
 # This goes directly cluster-> s3, doesn't bounce to local
 outputs_s3 = rh.folder(system=gpu, path="dreambooth/outputs").to("s3", path="runhouse/dreambooth/outputs")
 outputs_s3.save("dreambooth_outputs")
+```
 
-# and later:
-rh.folder(name="dreambooth_outputs").to("here")
+And later, if I want to load down the folder in full:
+```python
+rh.Folder.from_name("dreambooth_outputs").to("here")
+```
 
-# Load a table in from anywhere (S3, GCS, local, etc)
-my_table = rh.table(system="gcs", path="my_bucket/my_table.parquet").to("here")
-# preprocess...
+We support interactive blob and table primitives too:
```

Comment:
The folder and function examples are simple and great, but this one is a bit harder to digest (for a README). I think it's borderline so we can leave it for now if you feel strongly, or we could just saying that we support blob and table primitives, between file systems, clusters, and local without the full code demo. (maybe eventually move this to a quickstart guide page instead of readme?)

Reply:
I tried to make it into a little bit more of a real-looking application, but I agree it could be more digestible. I'm going to push it for now and then prune it shortly.